python/perf-kernels/flash-attention.py

```diff
@@ -811,11 +834,17 @@ def _attn_bwd(Q, K, V, sm_scale,
     k = tl.load(K_block_ptr)
     v = tl.load(V_block_ptr)
 
+    if USE_ALIBI != 0:
```

Comment:
Can you make this a boolean? We had multiple values for the other bias because it could be either vector or matrix, and while we could handle strings AOTriton could not so we converted to numbers.

Reply:
done